common/helpers.go

```diff
@@ -200,19 +201,69 @@ func goRoutineID() int {
 	return id
 }
 
-func interfaceFromRemoteObject(remoteObject *cdpruntime.RemoteObject) (interface{}, error) {
-	if remoteObject.ObjectID != "" {
-		return nil, ErrUnexpectedRemoteObjectWithID
+func parseRemoteObjectPreview(op *cdpruntime.ObjectPreview, logger *logrus.Entry) (map[string]interface{}, error) {
+	obj := make(map[string]interface{})
+	if op.Overflow {
+		logger.Warn("object will be parsed partially")
```

Comment:
`obj` is an empty map at this point ;)

I considered passing the `ObjectID` of the original `RemoteObject` just for logging purposes, but that probably wouldn't be helpful. Logging `op` itself might be too noisy, but I'll see if I can pass the property name at least.

Reply:
Got it!

Reply:
Actually, the property name would always be empty in this case, since this will never recursively parse a nested object (due to that CDP limitation). So the only time this will be logged is for main / root objects, which don't have a property name since they're not part of another object.

So I don't see any way we could improve this message. [Here are the fields on `ObjectPreview`.](https://pkg.go.dev/github.com/chromedp/cdproto/runtime#ObjectPreview) `Type` will always be "object" and `Description` will be `Object`, `Subtype` might be useful, but it's always empty for anonymous objects... And the other fields are not useful as identifiers.

At the same time I wouldn't want to remove this as it gives a hint that whatever might be logged later would be incomplete.